src/main/java/ip/duke/Duke.java

```diff
@@ -47,51 +47,54 @@ public static void main(String[] args) {
         System.out.println("Hello from\n" + logo);
         greet();
         // Get user input
-            Scanner sc = new Scanner(System.in);
+            Scanner in = new Scanner(System.in);
             String input;
             boolean isBye = false;
             // conversions loop
             do {
                 System.out.printf("%nMasterOm : ");
-                input = sc.nextLine().trim();
-                drawLine();
+                input = in.nextLine().trim();
+                drawALine();
+
                 if (input.isEmpty()) {
-                    sc.nextLine();
+                    // flush buffer
+                    in.nextLine();
                     echoEmptyInput();
                 }
+
                 try {
                     isBye = echoBye(input);
                 } catch (DukeException e) {
                     System.out.println(e.getMessage());
                 }finally {
-                    drawLine();
+                    drawALine();
                 }
             } while (!isBye);
-            sc.close();
+
+            in.close();
     }
     // This method processes tasks and generates dialogues
-    private static boolean echoBye(String input) throws DukeException {
+    private static boolean echoBye(String command) throws DukeException {
         // Exit program
-        if (input.equals("bye")) {
+        if (command.equals("bye")) {
             bye();
             return true;
         } else {
             // Parse user inputs, output corresponding tasks
             String[] words = null;
             // Check multiple words presence in input before splitting into a string array
-            if(input.indexOf(" ") > 0){
-                words = input.split(" ", 2);
-                input = words[0].trim();
+            if(command.indexOf(" ") > 0){
+                words = command.split(" ", 2);
+                command = words[0].trim();
```

Comment:
Since we are using split(" ") string method to split up the words delimited by spaces, it will naturally remove spaces from each of the elements returned.

```
String wordOne = "Hello World"
String[] splitWords = wordOne.split(" ")
// splitWords[0] == "Hello" and splitWords[1] == "World"

// In the case of multiple spaces (e.g. double spaces)
String wordTwo = "Hello  World"
splitWords = wordTwo.split(" ")
// splitWords[0] == "Hello"
// splitWords[1] == " "
// splitWords[2] == "World"
```

The trim() method removes whitespace from both ends of a string and since split handled that for you, you can save some extra steps here.